xl/xenops_server_xenlight.ml

```diff
@@ -2256,9 +2256,11 @@ module VM = struct
 						xs.Xs.setperms ent rwperm
 					)
 				) (
-					let dev_kinds = [ "vbd"; "vif"; "vfb"; "vkb"; "vfs"; "pci" ] in
 					[ "device"; "error"; "drivers"; "control"; "attr"; "data"; "messages"; "vm-data" ]
-					@ List.map (fun dev_kind -> "device/"^dev_kind) dev_kinds
+					@ if hvm then
+						let dev_kinds = [ "vbd"; "vif"; "vfb"; "vkb"; "vfs"; "pci" ] in
+						List.map (fun dev_kind -> "device/"^dev_kind) dev_kinds
+					else []
```

Comment:
Yes, it feels a bit unreasonable that Windows should require these nodes to be created for it.
The Windows people say it's necessary, but perhaps there's another way.